app/routes/wiki-page-error.js

```diff
@@ -12,7 +12,7 @@ export default Route.extend(
 		 */
 		renderTemplate(controller, error) {
 			if (!this.get('fastboot.isFastBoot')) {
-				logEvent('Wiki-page route error substate', error);
+				logError('Wiki-page route error substate', error);
```

Comment:
Do we still need this logging? if yes let's create a ticket to remove it in the future.

Reply:
I'd leave it in place - it gives us an overview of how many and in which places people are landing in error substates. Might become useful.